.buildkite/pipeline.full.yml

```
@@ -49,6 +65,26 @@ steps:
     concurrency_group: 'browserstack-app'
     concurrency_method: eager
 
+  - label: ':android: Debug fixture minimal tests'
+    depends_on: "fixture-debug"
+    timeout_in_minutes: 30
+    plugins:
+      artifacts#v1.2.0:
+        download: "build/debug/fixture-debug.apk"
+        upload: "maze_output/failed/**/*"
+      docker-compose#v3.7.0:
+        pull: android-maze-runner
+        run: android-maze-runner
+        command:
+          - "features/minimal"
```

```suggestion
          - "features/smoke_tests"
```

This will ensure that at least one NDK signal/exception gets raised, and hopefully flag up any issues with debug builds.